imports/server/accounts.ts

```diff
@@ -116,6 +116,7 @@ const DEFAULT_ENROLL_ACCOUNT_TEMPLATE = 'Hiya!\n' +
     'This message was sent to {{email}}';
 
 function makeView(user: Meteor.User | null, url: string) {
+  // eslint-disable-next-line jolly-roger/no-sync-mongo-methods
```

Comment:
Ah, and this is needed because Meteor's `Email` package doesn't yet support `sendAsync()`, nor do the various `Accounts.emailTemplates` generator functions get awaited at their invocations ( https://github.com/meteor/meteor/blob/b43018cdd6a20744869a3b1f24c7a4c115fbef64/packages/accounts-base/accounts_server.js#L1399-L1421 ).  While the former [got added in Meteor 2.9](https://github.com/meteor/meteor/pull/12101), we can't take 2.9 until 2.9.1 due to the fetch() type bug, and I don't see the latter fixed upstream yet, so yeah, better to punt on this for now.